.github/workflows/backwards_compatibility_marqo_orchestrator.yml

```diff
@@ -0,0 +1,176 @@
+# Orchestrator workflow
+name: Marqo Compatibility Tests Orchestrator
+
+on:
+  push:
+    branches:
+      - mainline
+    paths-ignore:
+      - '**.md'
+  workflow_dispatch:
+    inputs:
+      to_version:
+        description: 'Target Marqo version'
+        required: true
+      to_image:
```

Comment:
Can this accept a digest?

Reply:
This pipeline must be called with a digest, not try to retrieve it. Otherwise we run the same risks (that the tag isn't what we think it is). The digest must come out of the ECR workflow that pushes the image, so that we're confident we're testing the right image

Reply:
This now accepts a digest - I have completely removed the whole logic to retrieve digest using tags 
1. https://github.com/marqo-ai/marqo/pull/1024/commits/740097f76bc5fd67568d409c1a680d9ebcb77333#diff-f4c34005d6dcd764124973ffb9f157dd84675dec760dc24c981a4d02fec8a9daL134-L162
2. https://github.com/marqo-ai/marqo/pull/1024/commits/740097f76bc5fd67568d409c1a680d9ebcb77333#diff-a60798b1e43d7cc0bb54656f01ca73fc84579b6e501acc45d79cbfec3819cb1dR25-R28

Reply:
This will be a fully qualified image name after further discussion

Reply:
yep, I have changed it to that